src/lib/components/table/TableHeader.svelte

```diff
@@ -0,0 +1,31 @@
+<script lang="ts">
+  export let position: "top" | "left" | "top-left" = "top";
+  export let maxWidth = "";
+</script>
+
+<th
+  class="
+          p-0
+          sticky
+          {position === 'top' && 'top-0'}
```

Comment:
Is this the standard of switching class name we are using everywhere? Seems like a map would be better no?